src/storage/config.rs

```diff
@@ -57,6 +59,9 @@ impl Config {
         if self.data_dir != DEFAULT_DATA_DIR {
             self.data_dir = try!(config::canonicalize_path(&self.data_dir))
         }
+        if !self.raft_data_dir.is_empty() {
+            self.raft_data_dir = try!(config::canonicalize_path(&self.raft_data_dir))
```

Comment:
we must check data dir and raft data dir are not same, check data WAL dir and raft data WAL dir are not same.